php/libraries/DataFrameworkMenu.class.inc

```diff
@@ -183,5 +183,91 @@ abstract class DataFrameworkMenu extends NDB_Menu_Filter
         return new \LORIS\Http\DataIteratorBinaryStream($rows);
 
     }
+
+    /**
+     * Format the list of user's projects to display the module filter options
+     *
+     * @param bool $numericKeys If the returned projects keys
+     *                          should be numeric (ProjectID) or project name
+     *
+     * @return array - an associative array of the form (ID|mame => name)
+     */
+    public function getProjectOptions(
+        bool  $numericKeys = true
```

Comment:
Why does this option exist? (Same with on the helper functions).. it doesn't seem to ever be true.

Reply:
Some module filters works with index keys (half uses indexes, half names):
- modules/mri_violations/php/mri_violations.class.inc
- modules/mri_violations/php/resolved_violations.class.inc
- modules/behavioural_qc/php/behavioural.class.inc
- modules/behavioural_qc/php/conflicts.class.inc
- modules/behavioural_qc/php/incomplete.class.inc
- modules/media/php/media.class.inc
- modules/new_profile/php/new_profile.class.inc
- modules/statistics/php/stats_behavioural.class.inc
- modules/statistics/php/stats_demographic.class.inc
- modules/statistics/php/stats_mri.class.inc
- modules/user_accounts/php/edit_user.class.inc